flutter_local_notifications/README.md

```diff
@@ -180,7 +355,7 @@ await flutterLocalNotificationsPlugin.schedule(
     platformChannelSpecifics);
 ```
 
-Note that on Android devices, the default behaviour is that the notification may not be delivered at the specified time when the device in a low-power idle mode. This behaviour can be changed by setting the optional parameter named `androidAllowWhileIdle` to true when calling the `schedule` method.
+For Android devices, the default behaviour is that the notification may not be delivered at the specified time when the device in a low-power idle mode. This behaviour can be changed by setting the optional parameter named `androidAllowWhileIdle` to true when calling the `schedule` method.
```

Comment:
Change "For Android devices" to "On Android devices"

Reply:
It has been changed